cmake/emu/adsp.cmake

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
this is mis-using the emulation support for actually loading images into actual hardware. We should not do this.

Reply:
Thanks. fixed it.